nodejs/packages/layer/src/wrapper.ts

```diff
@@ -121,8 +122,12 @@ async function initializeProvider() {
   tracerProvider.register(sdkRegistrationConfig);
 
   // Configure default meter provider (doesn't export metrics)
+  const metricExporter = new OTLPMetricExporter();
   let meterConfig: MeterProviderOptions = {
     resource,
+    readers: [new PeriodicExportingMetricReader({
```

Comment:
@stevemao I think we need to import `PeriodicExportingMetricReader` from '@opentelemetry/sdk-metrics'

Reply:
fixed

Reply:
I think the reader should be added only if the `configureMeterProvider()` function is not defined.